tests/test_generate_data.py

```diff
@@ -65,3 +66,79 @@ def test_params_same_from_sim_data(self):
                  grouped_data['n_custs'])
         assert ((np.array(list(bbgb_params.values())) - np.array(
             bbtf._unload_params('alpha', 'beta', 'gamma', 'delta'))) < 0.1).all()
+
+
+bg_params = OrderedDict([('T', 100), ('r', 0.24), ('alpha', 4.41),  ('a', 0.79),
```

Comment:
can you turn these into pytest fixtures? Here's a quick explanation of them: https://docs.pytest.org/en/latest/fixture.html and I use them in other areas like https://github.com/CamDavidsonPilon/lifetimes/blob/47c911a705242bbbed258372c2f06e6afcbfd428/tests/test_utils.py#L12-L59